### PR TITLE
Merge some simple class' type stubs back to source

### PR DESCRIPTION
**Pull request by trim21**
some classes with less equal than 5 properties

### CODE REVIEW

File: github/BranchProtection.py [hunk 1]
```diff
@@ -20,11 +20,22 @@
...
+from github.PaginatedList import PaginatedList
+
+if TYPE_CHECKING:
+    from github.NamedUser import NamedUser
+    from github.RequiredPullRequestReviews import RequiredPullRequestReviews
+    from github.RequiredStatusChecks import RequiredStatusChecks
+    from github.Team import Team
 
 
 class BranchProtection(github.GithubObject.CompletableGithubObject):
```
Comment (EnricoMi) [inline github/BranchProtection.py:41]: types of attributes are not defined for this class

File: github/CommitStats.py [hunk 2]
```diff
@@ -36,24 +36,15 @@ class CommitStats(github.GithubObject.NonCompletableGithubObject):
     """
```
Comment (EnricoMi) [inline github/CommitStats.py:36]: types of attributes are not defined for this class

File: github/GitAuthor.py [hunk 3]
```diff
@@ -40,24 +41,15 @@ def __repr__(self):
         return self.get__repr__({"name": self._name.value})
```
Comment (EnricoMi) [inline github/GitAuthor.py:41]: types of attributes are not defined for this class

File: github/GitBlob.py [hunk 4]
```diff
@@ -40,42 +40,27 @@ def __repr__(self):
         return self.get__repr__({"sha": self._sha.value})
```
Comment (EnricoMi) [inline github/GitBlob.py:41]: types of attributes are not defined for this class

File: github/GitRef.py [hunk 5]
```diff
@@ -41,50 +49,46 @@ def __repr__(self):
         return self.get__repr__({"ref": self._ref.value})
```
Comment (EnricoMi) [inline github/GitRef.py:50]: types of attributes are not defined for this class

**Review by EnricoMi — CHANGES_REQUESTED** [review]
Types of attributes are not defined for many classes. Please turn this into a `Draft` until you are finished.

**Comment by trim21** [comment]
Types of attributes are not realy required in type system here...

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Pull request merged**